qkeras/qconvolutional.py

```diff
@@ -127,8 +128,11 @@ def __init__(self,
         bias_constraint=bias_constraint,
         **kwargs)
 
-  def call(self, inputs):
+  def call(self, inputs, training=None):
```

Comment:
to use "def call(self, inputs, is_training=False):"

Reply:
I used None since that is used in the Keras Dropout layer code. My guess is they did that because there exists the possibility of the training value not being passed? If so, we wouldn't want to default to False.
`  def call(self, inputs, training=None):
    if training is None:
      training = K.learning_phase()`
This is keras.layers.core.Dropout line 198
Thoughts?

Reply:
I see. 

BTW we need to refer to tf.keras
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/keras/layers/core.py#L211
instead of standalone keras.